reframe/frontend/executors/policies.py

```diff
@@ -116,15 +94,36 @@ def on_task_exit(self, task):
         pass
 
     def on_task_failure(self, task):
-        mssg = f'{task.check.info()} {format_duration(task.duration)}'
+        timings = task.pipeline_timings(['compile_complete',
+                                         'run_complete',
+                                         'total'])
+        mssg = f'{task.check.info()} [{timings}]'
```

Comment:
I don't like very much that we repeat this thing three times.

Reply:
We could have pipeline_timings get just an argument `verbose: boolean` and return the normal/verbose message instead of passing the list.